### ISSUE TITLE
Benchmark suite with LB for single-threaded servers

### ISSUE THREAD

**Issue by LesnyRumcajs** [LesnyRumcajs/grpc_bench#77]
Based on @brunoborges idea in https://github.com/LesnyRumcajs/grpc_bench/pull/76

Create a benchmark suite (ideally using the existing images) that would provide single-threaded implementations with a LB (to be chosen) so that the we can compare e.g. `rust_tonic_st` x 2 + LB against single dotnet appliance, both running on e.g. 3 cores.

**Comment by brunoborges** [comment]
Initially my comment was that the total amount of CPU/RAM for Single-Threaded Workload X plus LB should equal Multi-Threaded Workload Y minus LB.

But in reality, any cloud-based deployment would have a LB anyways. 

So the rule should probably be that the number of single-threaded instances should equal to # of CPUs given to one multi-threaded instance.

Benchmark results should only show (published) comparisons of 2 or more CPUs. 

The use of a single CPU should be limited to CICD validation.

**Comment by brunoborges** [comment]
And regarding `duration`, anything below 10min IMO is not realistic enough for fair comparisons between languages/runtimes, because some have JIT compilers.

**Comment by LesnyRumcajs** [comment]
@brunoborges I agree, it makes sense to split those. We even added `_mt` and `_st` suffix, but it should be kind of a different category.

**Comment by berestovskyy** [comment]
IMO nowadays in most cases load balancers come as an infrastructure, so I'm not sure `N * ST + LB` benchmarks will add much value...

Either you build a big service running on multiple cores, or you build a micro service targeting 1 CPU core, you can make your choice using the results we have at the moment...
